cv.tex

```diff
@@ -150,102 +149,70 @@ \subsection{Programming}
 
 \section{Experience}
 
-\runsubsection{Coursera}
-\descript{| KPCB Fellow + Software Engineering Intern}
+\runsubsection{Dropbox}
+\descript{Sr. SecEng (IC4), Detections \& Response Team}
 
-\location{Expected June 2014 – Sep 2014 | Mountain View, CA}
+\location{Nov 2022 – Nov 2024 | Seattle, WA}
 \vspace{\topsep} % Hacky fix for awkward extra vertical space
 \begin{tightitemize}
-\item 52 out of 2500 applicants chosen to be a KPCB Fellow 2014.
+\item Lead Incident Responder for the Detections and Response Team (DART), serving two public security incidents \textbf{\href{https://dropbox.tech/security/changing-how-we-identify-malicious-urls-in-shared-documents}{[1 - Malicious Links]}} \textbf{\href{https://sign.dropbox.com/blog/a-recent-security-incident-involving-dropbox-sign}{[2 - Dropbox Sign]}}
+\item Transitioned security case management operations and metrics from Jira to Tines
+\item Co-developed and monitored insider risk detections for multiple Reduction in Force (RIF) events
+\item Lead emergent issues with Audit, Compliance, Risk teams, documenting playbooks and interaction outcomes
+\item Mentored several great security engineers into their first enterprise incident response role
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
 
-\runsubsection{Google}
-\descript{| Software Engineering Intern}
+\runsubsection{Amazon.com}
```

Comment:
(JC):
- “Detections & Monitoring” —> “Detection & Monitoring”
- Consider: “Founded a team producing security alarms from GuardDuty, CloudTrail, and Linux endpoint telemetry sources” —> “Participated in founding the threat detection team focused on protecting the business’ AWS environments, leveraging AWS CloudTrail, Amazon GuardDuty, and process and network telemetry from Linux hosts”
- Consider this for your second bullet. Cherry pick. “Designed, developed, and operated a cloud-native detection system to analyze and enrich security logs spanning hundreds of thousands of AWS accounts to produce actionable threat detections.” Avoid “deprecated.” I understand what you might be going for there but all of that in a mattered a couple years may not be perceived as positive and the reader won’t have the context needed to understand why.
- “detections service into” —> “detection system with” or “service with” and change “communicate” to “convey”
- For standardizing documentation, why mention ADS at all? I think the key is “standardizing” and “documentation” such that there is a positive cross-team impact
(SIRT, Red Team, etc.) in know what and how we detect. The ADS part isn’t key.
- I don’t see mention around the day-to-day (i.e., detector development), partnering with SIRT to monitor, etc. Consider adding something.
- “Detections workstream lead” —> “Led a threat hunting workstream”
- I wouldn’t link to the Log4Shell hotpatch article. I think it’s misleading since we didn’t participate in delivering that. I wouldn’t hyperlink at all. For Twitch, rather than hyperlinking the “Twitch” add a “[Blog]” or something hyperlink like you did for Dropbox.
- How important is listing when you were promoted?

Reply:
1. ty
2. split the difference to save some line length, ty
3. ty
4. agree, ty
5. ADS by name is used by other enterprises, thinking of it like "standardized cyber risk modelling using the FAIR-CAM framework", tool used for a job that your shop might already be using. dropped the abbreviation since its not used again. The reader doesn't nessisarily know the full set of consuming teams of this documentation, will try to differentiate that
6. good feedback, leaving it for now.
7. bless ty, such a cleaner way to say it
8. ACK, i think i'd rather move the dropbox items to this format. swapped the AWS for a CISA link.
9. see above